### PR TITLE
Feat/rf 1394 implement stats page

### PR DESCRIPTION
**Pull request by nikaaru**
implement statistic page

### CODE REVIEW

File: components/statistics/StatisticChart/BarChart.tsx
Comment (mikasackermn) [inline components/statistics/StatisticChart/BarChart.tsx:1]: Screenshot from 2024-04-22 23-41-14
Also, the chart grids aren't compatible with Figma

File: components/statistics/TopList/index.tsx
Comment (mikasackermn) [inline components/statistics/TopList/index.tsx:1]: Screenshot from 2024-04-22 23-02-12
In my opinion, it's better to display - or zero in place of the value if there is null.

File: components/statistics/StatisticChart/BarChart.tsx
Comment (mikasackermn) [reply components/statistics/StatisticChart/BarChart.tsx:1]: Screenshot from 2024-04-27 13-09-23
This isn't fixed yet

**Comment by mikasackermn** [comment]
In responsive, everything is messed up. I am mindful that we lack the necessary design. However, I believe that everything should at least not be messed up in responsive

File: components/statistics/StatisticChart/BarChart.tsx
Comment (nikaaru) [reply components/statistics/StatisticChart/BarChart.tsx:1]: Done

File: components/statistics/TopList/index.tsx
Comment (nikaaru) [reply components/statistics/TopList/index.tsx:1]: Done

**Review by mikasackermn — APPROVED** [review]

**Comment by RanGojo** [comment]
- [ ] I think we could use small size tooltip for chart only if breakdown is selected.
- [ ] I think tooltip position for chart is confusing. it's not clear tooltip belongs to which data. 
- [ ] Please change volumes -> Volume in tooltip
- [ ] Please change transaction -> Transaction in tooltip
- [ ] I think clear button for select is not based on what we agree.
- [ ] When changing "break down" filter from "destination chain" to "None" there is a flicker with wrong data for chart.
- [ ] After merging with mikasa branch, I expect that top header will be removed. (like search page)

**Comment by nikaaru** [comment]
> * [x]  I think we could use small size tooltip for chart only if breakdown is selected. Please double check if it matches with design.
> * [ ]  I think tooltip position for chart is confusing. it's not clear tooltip belongs to which data. check this demo: https://airbnb.io/visx/barstack
> * [x]  Please change `volumes` -> `Volume` in tooltip
> * [x]  Please change `transaction` -> `Transactions` in tooltip
> * [x]  Please change `others` -> `Others`
> * [x]  I think clear button for select is not based on what we agree.
> * [ ]  When changing "break down" filter from "destination chain" to "None" there is a flicker with wrong data for chart.
> * [ ]  After merging with mikasa branch, I expect that top header will be removed. (like search page)



> * [x]  I think we could use small size tooltip for chart only if breakdown is selected. Please double check if it matches with design.
> * [ ]  I think tooltip position for chart is confusing. it's not clear tooltip belongs to which data. check this demo: https://airbnb.io/visx/barstack
> * [x]  Please change `volumes` -> `Volume` in tooltip
> * [x]  Please change `transaction` -> `Transactions` in tooltip
> * [x]  Please change `others` -> `Others`
> * [x]  I think clear button for select is not based on what we agree.
> * [ ]  When changing "break down" filter from "destination chain" to "None" there is a flicker with wrong data for chart.
> * [x]  After merging with mikasa branch, I expect that top header will be removed. (like search page)

I updated all of the above plus the colors.
I will only handle "When changing "break down" filter from "destination chain" to "None" there is a flicker with wrong data for chart." in task loading.

And about the trigger the tooltip, I spent a lot of time trying methods, but I did not find a solution, I also checked this link: [](https://airbnb.io/visx/barstack)
We are using the **@visx/xychart** library, the way the tooltip works is different, if it is essential, we can not use **@visx/xychart** and use Basic **Visx** library like this link: [](https://airbnb.io/visx/barstack)

**Review by RanGojo — APPROVED** [review]
looks good to me. good job.

**Pull request merged**